cli/crypto.py

```diff
@@ -1,30 +1,30 @@
 from dataclasses import dataclass
 from random import randint
-from typing import Tuple, List, Dict, Any
+from typing import Any, Dict, List, Tuple, cast
 
 from Crypto.Cipher import AES, PKCS1_OAEP
+from Crypto.Cipher._mode_eax import EaxMode
 from Crypto.PublicKey.RSA import RsaKey
 from Crypto.Random import get_random_bytes
 from eth2deposit.key_handling.keystore import Keystore, Pbkdf2Keystore
-from eth_typing import BLSPubkey, BLSSignature
-from py_ecc import optimized_bls12_381
-from py_ecc.bls import G2ProofOfPossession as bls_pop
+from eth_typing.bls import BLSPubkey, BLSSignature
+from py_ecc.bls.ciphersuites import G2ProofOfPossession as bls_pop
 from py_ecc.bls.g2_primitives import G2_to_signature, signature_to_G2
-from py_ecc.optimized_bls12_381 import Z2, add, multiply
+from py_ecc.optimized_bls12_381.optimized_curve import Z2, add, curve_order, multiply
 from py_ecc.utils import prime_field_inv
 
-PRIME = optimized_bls12_381.curve_order
+PRIME = curve_order
 
 
 @dataclass
-class HorcruxPbkdf2Keystore(Pbkdf2Keystore):
+class HorcruxPbkdf2Keystore(Pbkdf2Keystore):  # type: ignore
```

Comment:
eth2deposit has no type info shipped with it (no py.typed file) but mypy strict mode does not want to allow subclassing classes without type info.